src/database/storage/storage_schema/schema.rs

```diff
@@ -72,6 +72,7 @@ use crate::locks::tokio::LockCallbackFn;
 /// It works equally well to put them in a `struct`.  If the tables
 /// are all of the same type (including generics), they could be
 /// placed in a collection type such as `Vec`, or `HashMap`.
+#[derive(Debug, Clone)]
 pub struct DbtSchema {
```

Comment:
I'm not sure what `Debug` and `Clone` could be used for, or even means, for these structures. They also probably shouldn't be `pub` I think.

Reply:
Regarding `pub` visibility: #496.